packages/app/lib/utils/index.js

```diff
@@ -31,6 +31,44 @@ class FirebaseUtilsModule extends FirebaseModule {
     return this.native.isRunningInTestLab;
   }
 
+  get playServicesAvailability() {
+    if (isIOS) {
+      return {
+        isAvailable: true,
+        status: 0,
+      };
+    }
+    return this.native.androidPlayServices;
+  }
+
+  getPlayServicesStatus() {
+    if (isIOS) {
+      return Promise.resolve();
```

Comment:
`playServicesAvailability` above returns a default object on iOS, so lets do the same for this one, Promise.resolve the default object, rather than undefined - the user won't need to conditionally switch then based on platform

Types and the test above will need updating though also

Reply:
sure. I've updated the return type and the docs.